bin/reth/src/node/events.rs

```diff
@@ -37,22 +37,28 @@ impl NodeState {
     /// Processes an event emitted by the pipeline
     fn handle_pipeline_event(&mut self, event: PipelineEvent) {
         match event {
-            PipelineEvent::Running { stage_id, checkpoint } => {
+            PipelineEvent::Running { pipeline_progress, pipeline_total, stage_id, checkpoint } => {
                 let notable = self.current_stage.is_none();
                 self.current_stage = Some(stage_id);
                 self.current_checkpoint = checkpoint.unwrap_or_default();
 
                 if notable {
                     info!(
                         target: "reth::cli",
+                        pipeline_stages = %format!("{pipeline_progress}/{pipeline_total}"),
```

Comment:
this makes sense to me,

I'm unsure about the order here, but what we have:

`stages=y/x name progress checkpoint`

is the best order imo